src/components/Table/Table.jsx

```diff
@@ -219,7 +219,8 @@ class Table extends React.Component {
                 <MaterialTableRow>
                   {columns.map((column, colIndex) => (
                     <MaterialTableRowColumn key={`${colIndex}_sum`} style={{ color: column.color }}>
-                      {column.sumFn && (column.field !== 'life_state_dead' ?
+                      {column.sumFn && typeof column.sumFn === 'function' && abbreviateNumber(data.map(column.sumFn).reduce(sum, 0))}
```

Comment:
the only thing that differs is the part inside `data.map`.  Optimally the if condition would branch there so that we minimize duplicated code.

That might require moving this code out into a function outside the JSX so it's easier to write